src/components/scenes/FioAddressSettingsScene.js

```diff
@@ -6,20 +6,20 @@ import * as React from 'react'
 import { refreshAllFioAddresses } from '../../actions/FioAddressActions.js'
 import s from '../../locales/strings'
 import { FioActionSubmit } from '../../modules/FioAddress/components/FioActionSubmit'
-import { getTransferFee } from '../../modules/FioAddress/util'
+import { addBundles, getAddBundlesFee, getTransferFee } from '../../modules/FioAddress/util'
 import { connect } from '../../types/reactRedux.js'
 import { type NavigationProp, type RouteProp } from '../../types/routerTypes.js'
 import { SceneWrapper } from '../common/SceneWrapper'
 import { ButtonsModal } from '../modals/ButtonsModal'
-import { Airship, showError } from '../services/AirshipInstance'
+import { Airship, showError, showToast } from '../services/AirshipInstance'
 import { type ThemeProps, withTheme } from '../services/ThemeContext.js'
 import { EdgeText } from '../themed/EdgeText'
 import { MainButton } from '../themed/MainButton.js'
 import { Tile } from '../themed/Tile'
 
 type LocalState = {
+  showAddBundles: boolean,
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name